Rethinking_2/Chp_08.ipynb

```diff
@@ -0,0 +1,1578 @@
+{
```

Comment:
<p>add semicolon at the end of the last line to suppress output.</p><p> </p><p>standardised --&gt; standardized</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
I'm British... don't see anything wrong with that spelling :stuck_out_tongue: